modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/derived-config.txt

```diff
@@ -0,0 +1,586 @@
+com/liferay/frontend/taglib/clay/sample/web/internal/display/context/ClaySampleImageCard.lambda$getLabels$7(Lcom/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem;)V
```

Comment:
@chestofwonders Curious abouth what is the purpose of this file?

Reply:
Commited by mistake. It's generated by gradle